maintaining-and-tuning/monitoring.md

```diff
@@ -155,38 +172,42 @@ access to the port, for example by employing a firewall. Furthermore, read
+If you wish to disable or remove an alert, go to the dashboard and the corresponding panel, and there you can remove the alerting rule.
 
-7. You can now use the dashboards. The dashboards can give a general overview over the main components of the GRR server, which can be utilized by the user to monitor different metrics of each component. Examples for such metrics can be found in the [examples above](#example-queries). Remember that the dashboards and alerts are flexible, and can be expanded or modified to adjust to your exact needs. Additional metrics can be used by exploring `http://<host>:<port>/metrics` for each component of GRR server (change the port according to the GRR server component you want), and if you wish to create your own [custom dashboards](https://grafana.com/docs/grafana/latest/getting-started/getting-started/#create-a-dashboard), [panels](https://grafana.com/docs/grafana/latest/panels/add-a-panel/#add-a-panel) and [alerts](https://grafana.com/docs/grafana/latest/alerting/create-alerts/#create-alerts), make sure to go over the corresponding documentation in Grafana.
+1. You can now use the dashboards. The dashboards can give a general overview over the main components of the GRR server anf Fleetspeak servers, which can be utilized by the user to monitor different metrics of each component. Examples for such metrics can be found in the [examples above](#example-queries). Remember that the dashboards and alerts are flexible, and can be expanded or modified to adjust to your exact needs. Additional metrics can be used by exploring `http://<host>:<port>/metrics` for each component of GRR server (change the port according to the GRR server component you want) and Fleetspeak servers, and if you wish to create your own [custom dashboards](https://grafana.com/docs/grafana/latest/getting-started/getting-started/#create-a-dashboard), [panels](https://grafana.com/docs/grafana/latest/panels/add-a-panel/#add-a-panel) and [alerts](https://grafana.com/docs/grafana/latest/alerting/create-alerts/#create-alerts), make sure to go over the corresponding documentation in Grafana.
```

Comment:
Typo: anf -> and

Reply:
While we're at it, I'd probably replace "if you wish" with "if you'd like". It sounds simpler and I think I rarely saw "wish" in tech documentation.